src/components/Home/UpcomingEvents.tsx

```diff
@@ -421,5 +423,19 @@ export default styled(UpcomingEvents)`
 
 	.ant-picker-cell-in-view.ant-picker-cell-today .ant-picker-cell-inner::before {
 		border-radius: 50% !important;
+		border : 1.5px solid #e5007a;
+	}
+	.calenderDate{
+		margin-top : -24px;
+		background-color: #ff7ab4;
+		color : #fff;
+		border-radius : 50%;
+		display : flex;
+		align-items: center;
+		justify-content: center;
+		position: relative;
+	}
+	.calenderDate::before {
```

Comment:
why are we adding `.calenderDate::before`, it doesn't effect anything we can remove this

Reply:
Resolved, pls check